mlmicrophysics/models.py

```diff
@@ -140,16 +140,20 @@ def save_fortran_model(self, filename):
         nn_ds.to_netcdf(filename, encoding={'layer_names':{'dtype': 'S1'}})
         return
 
-    def predict(self, x):
+    def predict(self, x, batch_size=None):
+        if batch_size is None:
```

Comment:
Hi @djgagne  - just curious about why this is better practice? Thanks!

Reply:
The batch size for prediction can be much larger than the batch size for training, and its size depends on the computer being run on rather than being a hyperparameter of the model. I could specify a fixed value instead of None, but I wanted it to tie the default to the training batch_size since that is a reasonable first guess.